src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -129,6 +129,58 @@
           text: >
             az lock update --name lockName --resource-group group --notes newNotesHere --lock-type CanNotDelete
     """
+helps['account lock'] = """
+    type: group
+    short-summary: Manage Azure locks.
+"""
+helps['account lock create'] = """
+    type: command
+    short-summary: Create a subscription lock.
+    parameters:
+        - name: --notes
+          type: string
+          short-summary: Notes about this lock.
```

Comment:
To avoid redundancies(all will become liabilities), author the help at one place in the _params.py